README.rst

```diff
@@ -51,7 +51,7 @@ rooms, and mailing lists is expected to follow the `PyPA Code of Conduct`_.
 .. _`issue tracker`: https://github.com/pypa/warehouse/issues
 .. _`pypi.org`: https://pypi.org/
 .. _`on Freenode`: https://webchat.freenode.net/?channels=%23pypa-dev,pypa
-.. _`pypa-dev mailing list`: https://groups.google.com/forum/#!forum/pypa-dev
+.. _`pypa-dev mailing list`: https://mail.python.org/mailman3/lists/distutils-sig.python.org/
```

Comment:
A suggestion: since the new mailing list is called `distutils-sig` instead of `pypa-dev`, I think it would be helpful to readers if it said `distutils-sig mailing list` instead of `pypa-dev mailing list`.

Reply:
thanks